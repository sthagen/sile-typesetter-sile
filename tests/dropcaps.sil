\begin[papersize=a6]{document}
\nofolios
\use[module=packages.dropcaps]
\font[size=25pt]
\dropcap[lines=2]{T}esting one two three, don't collide with me.
Four five six, no crossing sticks.
\end{document}
