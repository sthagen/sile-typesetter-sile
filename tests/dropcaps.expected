Set paper size 	297.6377985	419.5275636
Begin page
Mx 	55.5086
My 	39.8973
Mx 	14.8819
My 	69.8973
Set font 	Gentium Plus;73.7619;400;;normal;;;LTR;
T	55 w=40.6267	(T)
Mx 	55.5086
My 	39.8973
Set font 	Gentium Plus;25;400;;normal;;;LTR;
T	72 86 87 76 81 74 w=62.5732	(esting)
Mx 	127.0178
T	82 81 72 w=37.9150	(one)
Mx 	173.8688
T	87 90 82 w=38.8306	(two)
Mx 	221.6353
T	87 75 85 72 72 w=55.3955	(three)
Mx 	277.0308
T	15 w=5.7251	(,)
Mx 	55.5086
My 	69.8973
T	71 82 81 10 87 w=54.2236	(don't)
Mx 	116.5563
T	70 82 79 79 76 71 72 w=68.3716	(collide)
Mx 	191.7521
T	90 76 87 75 w=46.8262	(with)
Mx 	245.4024
T	80 72 w=31.6284	(me)
Mx 	277.0308
T	17 w=5.7251	(.)
Mx 	14.8819
My 	99.8973
T	41 82 88 85 w=47.6685	(Four)
Mx 	71.3523
T	737 89 72 w=38.4277	(five)
Mx 	118.5821
T	86 76 91 w=29.0039	(six)
Mx 	147.5860
T	15 w=5.7251	(,)
Mx 	162.1131
T	81 82 w=26.3672	(no)
Mx 	197.2823
T	70 85 82 86 86 76 81 74 w=85.4736	(crossing)
Mx 	14.8819
My 	129.8973
T	86 87 76 70 78 86 w=57.8125	(sticks)
Mx 	72.6944
T	17 w=5.7251	(.)
End page
Finish
